src/site/antora/modules/ROOT/pages/manual/layouts.adoc

```diff
@@ -16,39 +16,25 @@
 ////
 = Layouts
 
-An Appender uses a Layout to format a LogEvent into a form that meets the needs of whatever will be consuming the log event.
-In Log4j 1.x and Logback Layouts were expected to transform an event into a String.
-In Log4j 2 Layouts return a byte array.
-This allows the result of the Layout to be useful in many more types of Appenders.
-However, this means you need to configure most Layouts with a
+An Appender uses a Layout to format a LogEvent into a form that meets
```

Comment:
Are there any changes in this file?

It seems it was only accidentally reformatted.